### ISSUE TITLE
[RE-TEST] Safari: Cannot navigate through the page using 'tab' key

### ISSUE THREAD

**Issue by Matejk00** [SAP/spartacus#13181]
**Describe the bug**
Cannot navigate through the storefront using 'tab' key. Focus locks on the header elements and there is no possibility to move further.

**Tell us the version of Spartacus**
Library version: 4.0-RC.0 deployed on http://spartacus-devci7672.eastus.cloudapp.azure.com:4200/electronics-spa/en/USD/

**To Reproduce**
1. Go to Spartacus Storefront homepage or any page
2. Try to focus elements one by one from the top using tab 'key'.
3. See the bug after pressing tab when last focusable element in header is focused. (Focus goes to Safari address bar and there is no possibility to move forward).

**Expected behavior**
Safari users can go through all focusable elements one by one pressing tab key.

**Desktop (please complete the following information):**
OS: macOS Big Sur 11.3.1
Browser: Safari Version 14.1 (16611.1.21.161.6)

**Comment by developpeurweb** [comment]
- Works fine for me on Safari Version 14.1.1 (16611.2.7.1.4)
- @Matejk00 can you please retest, but first make sure you have [this configuration in your Safari](https://github.com/SAP/spartacus/issues/5041#issuecomment-558883497), ignore the theme Calydon reference.

**Comment by Matejk00** [comment]
Re-tested with mentioned configuration, it's working fine.
